src/js/express.js

```diff
@@ -0,0 +1,148 @@
+// Express instance represented as a component
+//
+// Add additional routers and middleware as child components of this module
+"use strict";
+var fluid       = fluid || require("infusion");
+var namespace   = "gpii.express";
+var express     = fluid.registerNamespace(namespace);
+var path        = require("path");
+var exphbs      = require("express-handlebars");
+var handlebars  = require("handlebars");
+
+express.initExpress = function(that) {
+    if (!that.options.config || !that.options.config.express) {
+        console.error("Cannot initialize express because you have not supplied a 'config' option.");
+        return;
+    }
+
+    // We need these to be model variables so that other grades can listen and take action when they are changed.
+    var express  = require("express");
```

Comment:
Understood, that and previous comments have helped clarify the proper use of the model.  I have moved the express and router variables to become top-level elements and eliminated the use of change appliers.